diff --git a/config/alacritty/alacritty.toml b/config/alacritty/alacritty.toml
index d21e9ab..0fff2a8 100644
--- a/config/alacritty/alacritty.toml
+++ b/config/alacritty/alacritty.toml
@@ -20,22 +20,22 @@ family = "monospace"
 [[keyboard.bindings]]
 action = "Paste"
 key = "V"
-mods = "Control|Shift"
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "Copy"
 key = "C"
-mods = "Control|Shift"
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "IncreaseFontSize"
-key = "Minus"
+key = "."
 mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "DecreaseFontSize"
-key = "Minus"
-mods = "Control"
+key = ","
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "SpawnNewInstance"
@@ -44,12 +44,12 @@ mods = "Alt"
 
 [[keyboard.bindings]]
 action = "ScrollLineDown"
-key = "J"
+key = "N"
 mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "ScrollLineUp"
-key = "K"
+key = "E"
 mods = "Control|Alt"
 
 [scrolling]
diff --git a/config/hypr/hyprland.conf b/config/hypr/hyprland.conf
index 8c78a0c..df66d9a 100644
--- a/config/hypr/hyprland.conf
+++ b/config/hypr/hyprland.conf
@@ -143,51 +143,48 @@ $gcs = SUPER + SHIFT + CONTROL
 $gac = SUPER + ALT + CONTROL
 $gacs = SUPER + ALT + CONTROL + SHIFT
 
-bind = $gac,n,exec, alacritty
-bind = $gacs,n,exec, alacritty -e btop
-
 bind = $g, y, togglefloating
 
-bind = $g, backspace, exec, plsdo keyboard_layout choose
+bind = $g, backspace, exec, plsdo keyboard_layout choose # TODO: probably dont need?
 bind = $g, n, exec, plsdo workspace open_pinned notes
 bind = $g, m, exec, plsdo workspace open_pinned ncmpcpp
-bind = $g, f, exec, plsdo workspace open_pinned newsboat
-bind = $g, w, exec, plsdo workspace open_pinned Firefox
-bind = $g, v, exec, plsdo workspace open_pinned pulsemixer
-bind = $g, c, exec, plsdo workspace open_pinned btop
+bind = $g, l, exec, plsdo workspace open_pinned newsboat
+bind = $g, t, exec, plsdo workspace open_pinned Firefox
+bind = $g, h, exec, plsdo workspace open_pinned pulsemixer
+bind = $g, u, exec, plsdo workspace open_pinned btop
 bind = $g, o, exec, zo -m gui
 bind = $g, return, exec, alacritty
 bind = $g, q, killactive,
-bind = $gs, q, exec, hyprkill
-bind = $gs, s, exec, plsdo power
+bind = $gacs, q, exec, hyprkill
+bind = $gac, q, exec, plsdo power
 bind = $g, p, exec, j4-dmenu-desktop --dmenu='bemenu --prompt Launch'
-bind = $gcs, bracketleft, pseudo, # dwindle
-bind = $gcs, bracketright, togglesplit, # dwindle
-bind = $g, semicolon, fullscreenstate,2 2
-bind = $gs, semicolon, fullscreenstate,-1 2
-bind = $gcs, z, exec, sleep 1 && hyprctl dispatch dpms off # probably unused 
-bind = $gcs, slash, exec, sleep 1 && hyprctl dispatch dpms on # probably unused
-bind = $gs, period, exec, hyprpicker | cliphist store
-bind = $gs, comma, exec, cliphist list | sort -rn | bemenu --prompt 'Clipboard' -l 24 | cliphist decode | wl-copy
+bind = $g, quote, pseudo,
+bind = $g, w, togglesplit,
+bind = $g, f, fullscreenstate,2 2
+bind = $gs, f, fullscreenstate,-1 2
+bind = $gac, w, exec, sleep 1 && hyprctl dispatch dpms off
+bind = $gacs, w, exec, sleep 1 && hyprctl dispatch dpms on
+bind = $g, k, exec, hyprpicker | cliphist store
+bind = $g, c, exec, cliphist list | sort -rn | bemenu --prompt 'Clipboard' -l 24 | cliphist decode | wl-copy
 bind = $g, slash, exec, busctl --user call rs.wl-gammarelay / rs.wl.gammarelay ToggleInverted
-bind = ,pause,exec,echo 1 >> /tmp/eww-show-all && eww open cal # probably unused
-bindr = ,pause,exec,echo 0 >> /tmp/eww-show-all && eww close cal # probably unused
-bind = $gs, b, exec, [float] pidof bluetui || alacritty -e bluetui
-bind = $gs, t, exec, [float] transmission-list
-
-bind = $g, h, movefocus, l
-bind = $g, j, movefocus, d
-bind = $g, k, movefocus, u
-bind = $g, l, movefocus, r
-bind = $gs, h, movewindow, l
-bind = $gs, j, movewindow, d
-bind = $gs, k, movewindow, u
-bind = $gs, l, movewindow, r
-bind = $a, tab, cyclenext
-bind = $g + CONTROL, h, exec, plsdo workspace focus prev-current
-bind = $g + CONTROL, l, exec, plsdo workspace focus next-current
-bind = $gcs, h, exec, plsdo workspace move prev-current
-bind = $gcs, l, exec, plsdo workspace move next-current
+bind = $g, b, exec, [float] pidof bluetui || alacritty -e bluetui
+
+bind = $g, s, exec, screenshot
+bind = $gs, s, exec, screenshot full
+
+bind = $gac , m, movefocus, l
+bind = $gac , n, movefocus, d
+bind = $gac , e, movefocus, u
+bind = $gac , i, movefocus, r
+bind = $gacs, m, movewindow, l
+bind = $gacs, n, movewindow, d
+bind = $gacs, e, movewindow, u
+bind = $gacs, i, movewindow, r
+bind = $gac , f, exec, plsdo workspace focus prev-current
+bind = $gac , o, exec, plsdo workspace focus next-current
+bind = $gacs, f, exec, plsdo workspace move prev-current
+bind = $gacs, o, exec, plsdo workspace move next-current
+bind = $a, tab, cyclenext # TODO: better alt-tab behavior (task)
 
 bind = $g, 1, exec, plsdo workspace focus id 1
 bind = $g, space, exec, plsdo workspace focus id 1
@@ -200,7 +197,6 @@ bind = $g, 7, exec, plsdo workspace focus id 7
 bind = $g, 8, exec, plsdo workspace focus id 8
 bind = $g, 9, exec, plsdo workspace focus id 9
 bind = $g, 0, exec, plsdo workspace focus id 10
-bind = $g, a, exec, screenshot
 
 bind = $gs, 1, exec, plsdo workspace move id 1
 bind = $gs, space, exec, plsdo workspace focus id 1
@@ -213,54 +209,53 @@ bind = $gs, 7, exec, plsdo workspace move id 7
 bind = $gs, 8, exec, plsdo workspace move id 8
 bind = $gs, 9, exec, plsdo workspace move id 9
 bind = $gs, 0, exec, plsdo workspace move id 10
-bind = $gs, a, exec, screenshot full
 
 bind = $g, mouse_down, workspace, e+1
 bind = $g, mouse_up, workspace, e-1
 
-bind = $g,bracketleft,exec,plsdo audio volume set -r -5
-bind = $g,backslash,exec,plsdo audio volume toggle-mute
-bind = $gs,m,exec,plsdo audio output choose
-bind = $g,bracketright,exec,plsdo audio volume set -r +5
-bind = $gs,minus,exec,mpc seekthrough -10
-bind = $gs,bracketleft,exec,mpc prev
-bind = $gs,backslash,exec,mpc toggle
-bind = $gs,bracketright,exec,mpc next
-bind = $gs,equal,exec,mpc seekthrough +10
-
-bind = $g,b,submap,resize
+bind = $gs,a,exec,plsdo audio output choose
+bind = ,xf86audiolowervolume,exec,plsdo audio volume set -r -5
+bind = ,xf86audiomute,exec,plsdo audio volume toggle-mute
+bind = ,xf86audioraisevolume,exec,plsdo audio volume set -r +5
+bind = ,xf86audiorewind,exec,mpc seekthrough -10
+bind = ,xf86audioprev,exec,mpc prev
+bind = ,xf86audioplay,exec,mpc toggle
+bind = ,xf86audionext,exec,mpc next
+bind = ,xf86audioforward,exec,mpc seekthrough +10
+
+bind = $g,z,submap,resize
 submap = resize
-binde = ,h,resizeactive,-10 0
-binde = ,j,resizeactive,0 10
-binde = ,k,resizeactive,0 -10
-binde = ,l,resizeactive,10 0
+binde = ,m,resizeactive,-10 0
+binde = ,n,resizeactive,0 10
+binde = ,e,resizeactive,0 -10
+binde = ,i,resizeactive,10 0
 bind = ,escape,submap,reset
 submap = reset
 
 bind = $g,x,submap,swap
 submap = swap
-bind = ,h,swapwindow,l
-bind = ,h,submap,reset
-bind = ,j,swapwindow,d
-bind = ,j,submap,reset
-bind = ,k,swapwindow,u
-bind = ,k,submap,reset
-bind = ,l,swapwindow,r
-bind = ,l,submap,reset
+bind = ,m,swapwindow,l
+bind = ,m,submap,reset
+bind = ,n,swapwindow,d
+bind = ,n,submap,reset
+bind = ,e,swapwindow,u
+bind = ,e,submap,reset
+bind = ,i,swapwindow,r
+bind = ,i,submap,reset
 bind = ,escape,submap,reset
 submap = reset
 
 bind = $g,comma,submap,brightness
 submap = brightness
-binde = ,j,exec,plsdo brightness -d down 5
-binde = ,k,exec,plsdo brightness -d up 5
+binde = ,n,exec,plsdo brightness -d down 5
+binde = ,e,exec,plsdo brightness -d up 5
 bind = ,escape,submap,reset
 submap = reset
 
 bind = $g,period,submap,gamma
 submap = gamma
-binde = ,j,exec,plsdo colortemp -d down 200
-binde = ,k,exec,plsdo colortemp -d up 200
+binde = ,n,exec,plsdo colortemp -d down 200
+binde = ,e,exec,plsdo colortemp -d up 200
 bind = ,escape,submap,reset
 submap = reset
  
diff --git a/config/lesskey b/config/lesskey
index 7a18134..8a1988c 100644
--- a/config/lesskey
+++ b/config/lesskey
@@ -1,9 +1,9 @@
-j forw-line
-k back-line
-J forw-line-force
-K back-line-force
-h left-scroll
-l right-scroll
-n repeat-search
-N reverse-search
+n forw-line
+e back-line
+N forw-line-force
+E back-line-force
+m left-scroll
+i right-scroll
+j repeat-search
+J reverse-search
 q quit
diff --git a/config/yazi/keymap.toml b/config/yazi/keymap.toml
index 4d5ed51..0294331 100644
--- a/config/yazi/keymap.toml
+++ b/config/yazi/keymap.toml
@@ -7,3 +7,67 @@ desc = "Hide or show preview"
 on = ["c", "a"]
 run = "plugin compress"
 desc = "Archive selected files"
+
+[[manager.prepend_keymap]]
+on = "m"
+run = "leave"
+
+[[manager.prepend_keymap]]
+on = "n"
+run = "arrow 1"
+
+[[manager.prepend_keymap]]
+on = "e"
+run = "arrow -1"
+
+[[manager.prepend_keymap]]
+on = "i"
+run = "enter"
+
+[[manager.prepend_keymap]]
+on = "h"
+run = "find_arrow"
+
+[[manager.prepend_keymap]]
+on = "M"
+run = "back"
+
+[[manager.prepend_keymap]]
+on = "N"
+run = "seek 5"
+
+[[manager.prepend_keymap]]
+on = "E"
+run = "seek -5"
+
+[[manager.prepend_keymap]]
+on = "I"
+run = "forward"
+
+[[manager.prepend_keymap]]
+on = "H"
+run = "find_arrow --previous"
+
+[[manager.prepend_keymap]]
+on = ["l", "s"]
+run = "linemode size"
+
+[[manager.prepend_keymap]]
+on = ["l", "p"]
+run = "linemode permissions"
+
+[[manager.prepend_keymap]]
+on = ["l", "b"]
+run = "linemode btime"
+
+[[manager.prepend_keymap]]
+on = ["l", "m"]
+run = "linemode mtime"
+
+[[manager.prepend_keymap]]
+on = ["l", "o"]
+run = "linemode owner"
+
+[[manager.prepend_keymap]]
+on = ["l", "n"]
+run = "linemode none"
diff --git a/config/zathura/zathurarc b/config/zathura/zathurarc
index ebb8348..c8aa145 100644
--- a/config/zathura/zathurarc
+++ b/config/zathura/zathurarc
@@ -36,3 +36,49 @@ set recolor-darkcolor           "#e0def4"
 
 set recolor                     "false"
 set recolor-keephue             "false"
+
+# keymaps
+
+map t follow
+
+map m scroll left
+map n scroll down
+map e scroll up
+map i scroll right
+
+map N navigate next
+map E navigate previous
+
+map j search forward
+map J search backward
+
+map k mark_add
+
+map f toggle_fullscreen
+
+# Fullscreen bindings are the same, but have to be repeated 
+map [fullscreen] t follow
+
+map [fullscreen] m scroll left
+map [fullscreen] n scroll down
+map [fullscreen] e scroll up
+map [fullscreen] i scroll right
+
+map [fullscreen] N navigate next
+map [fullscreen] E navigate previous
+
+map [fullscreen] j search forward
+map [fullscreen] J search backward
+
+map [fullscreen] k mark_add
+
+map [fullscreen] f toggle_fullscreen
+
+# Index 
+map [index] m navigate_index collapse
+map [index] n navigate_index down
+map [index] e navigate_index up
+map [index] i navigate_index expand
+
+map [index] M navigate_index collapse-all
+map [index] I navigate_index expand-all
