diff --git a/config/alacritty/alacritty.toml b/config/alacritty/alacritty.toml
index d21e9ab..0fff2a8 100644
--- a/config/alacritty/alacritty.toml
+++ b/config/alacritty/alacritty.toml
@@ -20,22 +20,22 @@ family = "monospace"
 [[keyboard.bindings]]
 action = "Paste"
 key = "V"
-mods = "Control|Shift"
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "Copy"
 key = "C"
-mods = "Control|Shift"
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "IncreaseFontSize"
-key = "Minus"
+key = "."
 mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "DecreaseFontSize"
-key = "Minus"
-mods = "Control"
+key = ","
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "SpawnNewInstance"
@@ -44,12 +44,12 @@ mods = "Alt"
 
 [[keyboard.bindings]]
 action = "ScrollLineDown"
-key = "J"
+key = "N"
 mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "ScrollLineUp"
-key = "K"
+key = "E"
 mods = "Control|Alt"
 
 [scrolling]
diff --git a/config/lesskey b/config/lesskey
index 7a18134..8a1988c 100644
--- a/config/lesskey
+++ b/config/lesskey
@@ -1,9 +1,9 @@
-j forw-line
-k back-line
-J forw-line-force
-K back-line-force
-h left-scroll
-l right-scroll
-n repeat-search
-N reverse-search
+n forw-line
+e back-line
+N forw-line-force
+E back-line-force
+m left-scroll
+i right-scroll
+j repeat-search
+J reverse-search
 q quit
diff --git a/config/zathura/zathurarc b/config/zathura/zathurarc
index ebb8348..c8aa145 100644
--- a/config/zathura/zathurarc
+++ b/config/zathura/zathurarc
@@ -36,3 +36,49 @@ set recolor-darkcolor           "#e0def4"
 
 set recolor                     "false"
 set recolor-keephue             "false"
+
+# keymaps
+
+map t follow
+
+map m scroll left
+map n scroll down
+map e scroll up
+map i scroll right
+
+map N navigate next
+map E navigate previous
+
+map j search forward
+map J search backward
+
+map k mark_add
+
+map f toggle_fullscreen
+
+# Fullscreen bindings are the same, but have to be repeated 
+map [fullscreen] t follow
+
+map [fullscreen] m scroll left
+map [fullscreen] n scroll down
+map [fullscreen] e scroll up
+map [fullscreen] i scroll right
+
+map [fullscreen] N navigate next
+map [fullscreen] E navigate previous
+
+map [fullscreen] j search forward
+map [fullscreen] J search backward
+
+map [fullscreen] k mark_add
+
+map [fullscreen] f toggle_fullscreen
+
+# Index 
+map [index] m navigate_index collapse
+map [index] n navigate_index down
+map [index] e navigate_index up
+map [index] i navigate_index expand
+
+map [index] M navigate_index collapse-all
+map [index] I navigate_index expand-all
