diff --git a/config/alacritty/alacritty.toml b/config/alacritty/alacritty.toml
index d21e9ab..0fff2a8 100644
--- a/config/alacritty/alacritty.toml
+++ b/config/alacritty/alacritty.toml
@@ -20,22 +20,22 @@ family = "monospace"
 [[keyboard.bindings]]
 action = "Paste"
 key = "V"
-mods = "Control|Shift"
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "Copy"
 key = "C"
-mods = "Control|Shift"
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "IncreaseFontSize"
-key = "Minus"
+key = "."
 mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "DecreaseFontSize"
-key = "Minus"
-mods = "Control"
+key = ","
+mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "SpawnNewInstance"
@@ -44,12 +44,12 @@ mods = "Alt"
 
 [[keyboard.bindings]]
 action = "ScrollLineDown"
-key = "J"
+key = "N"
 mods = "Control|Alt"
 
 [[keyboard.bindings]]
 action = "ScrollLineUp"
-key = "K"
+key = "E"
 mods = "Control|Alt"
 
 [scrolling]
diff --git a/config/lesskey b/config/lesskey
index 7a18134..8a1988c 100644
--- a/config/lesskey
+++ b/config/lesskey
@@ -1,9 +1,9 @@
-j forw-line
-k back-line
-J forw-line-force
-K back-line-force
-h left-scroll
-l right-scroll
-n repeat-search
-N reverse-search
+n forw-line
+e back-line
+N forw-line-force
+E back-line-force
+m left-scroll
+i right-scroll
+j repeat-search
+J reverse-search
 q quit
